src/main/java/blackjack/model/ShuffleCards.java

```diff
@@ -0,0 +1,27 @@
+package blackjack.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class ShuffleCards {
+    private static final List<Card> cards;
+
+    static{
+        cards = new ArrayList<>();
+        Arrays.stream(Denomination.values()).forEach(
+                denomination -> Arrays.stream(Suit.values()).forEach(
+                        suit -> cards.add(new Card(denomination, suit))
+                )
+        );
```

Comment:
stream을 이처럼 쓰면 가독성이 오히려 떨어진다고 생각해.
```suggestion
        Arrays.stream(Denomination.values())
        .forEach(denomination -> {
                    Arrays.stream(Suit.values())
                          .forEach(suit -> cards.add(new Card(denomination, suit)))
                }
        );
```